testing/py3-rply/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
```

Comment:
Proposed patch: http://ix.io/1TXT

- Uses py3-pytest to run tests
- Set noarch